### ISSUE TITLE
Integer keys produce invalid JSON

### ISSUE THREAD

**Issue by lluchs** [xp-forge/json#9]
Example test:

``` diff
diff --git a/src/test/php/text/json/unittest/JsonOutputTest.class.php b/src/test/php/text/json/unittest/JsonOutputTest.class.php
index 436e7fc..f6e8420 100755
--- a/src/test/php/text/json/unittest/JsonOutputTest.class.php
+++ b/src/test/php/text/json/unittest/JsonOutputTest.class.php
@@ -135,6 +135,7 @@ abstract class JsonOutputTest extends \unittest\TestCase {
   #[@test, @values([
   #  ['{"":"value"}', ['' => 'value']],
   #  ['{"key":"value"}', ['key' => 'value']],
+  #  ['{"123":456}', [123 => 456]],
   #  ['{"a":"v1","b":"v2"}', ['a' => 'v1', 'b' => 'v2']]
   #])]
   public function write_object($expected, $write) {
```

This is a regression for xp-framework/rest 7.2.0.

**Comment by thekid** [comment]
Good catch, thanks!

**Comment by thekid** [comment]
:shipit: Released in [2.1.1](https://github.com/xp-forge/json/releases/tag/v2.1.1)
